articles/cms/wordpress/configuration.md

```diff
@@ -83,6 +83,8 @@ First, we'll check for the Application created for your WordPress site.
 
 In order for your WordPress site to perform certain actions on behalf of your Auth0 tenant, you'll need to authorize the Application created above to access the Management API.
 
+1. Make sure your Application allows the Client Credential grant (step 11 above)
```

Comment:
I know this is duplicative but I often find myself linking directly to this section during support and this is a key step, often the one that's missed.